src/Config/DabConfigEvents.cs

```diff
@@ -14,4 +14,5 @@ public class DabConfigEvents
     public const string MYSQL_QUERY_EXECUTOR_ON_CONFIG_CHANGED = "MYSQL_QUERY_EXECUTOR_ON_CONFIG_CHANGED";
     public const string POSTGRESQL_QUERY_EXECUTOR_ON_CONFIG_CHANGED = "POSTGRESQL_QUERY_EXECUTOR_ON_CONFIG_CHANGED";
     public const string DOCUMENTOR_ON_CONFIG_CHANGED = "DOCUMENTOR_ON_CONFIG_CHANGED";
+    public const string AUTHZ_RESOLVER_ON_CONFIG_CHANGED = "AUTHZ_RESOLVER_ON_CONFIG_CHANGED";
 }
```

Comment:
Does this comment in the description still apply? 

`"However, there may be issue with metadataprovider hot-reload not updating all internal state properly."`

Reply:
no, the pr that was pending merge when that comment was written was merged. Updated PR description.